### PR TITLE
Add analytics dashboard

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
This PR 
1. Refactors the existing widgets to use dimension selector
2. Adds the new manager analytics and publisher analytics dashboards.
3. Add drill down across widgets

........................................................................................

**Publisher analytics**

_Overall Usage Page_
publisher_overall_usage

_API Usage Page_
publisher_api_use

_API Performance Page_
publisher_latency

_API Faults Page_
publisher_api_faults_1
publisher_api_faults_2

................................................................................................................

**Manager Dashboard**

_Overview Page_
manger_overview

_API and Application Page_
manger_apistats

_Developer Statistics Page_
manger_devstats

_Overall API Usage Page_
manger_ overallpng

_API Usage Page_
manger_ apiusagepng

_API Performance Page_
manger_ latencypng

_Top Faulty Page_
manger_ topfaultypng

_API Faults Page_
manger_apifaults2
manger_apifaults1

_Devices Page_
manger_devices

_Geo Page_
manger_geo

### CODE REVIEW

File: components/org.wso2.analytics.apim.widgets/APIMApiCreatedAnalytics/src/APIMApiCreatedAnalyticsWidget.jsx [hunk 1]
```diff
@@ -245,73 +244,82 @@ class APIMApiCreatedAnalyticsWidget extends Widget {
...
+            const { drillDown } = configs.options;
+
+            if (drillDown) {
+                const { apiname, apiversion } = data;
+                const api = (apiname.split(' (')[0]).trim();
+                const provider = (apiname.split('(')[1]).split(')')[0].trim();
+                const locationParts = window.location.pathname.split('/');
+                const dashboard = locationParts[locationParts.length - 2];
+
+                window.location.href = window.contextPath
```
[the remark this answers is not in the document]
Comment (ruwiniwj) [reply components/org.wso2.analytics.apim.widgets/APIMApiCreatedAnalytics/src/APIMApiCreatedAnalyticsWidget.jsx:318]: if we hardcode the dashboard name then this widget won't be reusable in custom dashboards.

[the remark this answers is not in the document]
Comment (pahans) [reply components/org.wso2.analytics.apim.widgets/APIMApiCreatedAnalytics/src/APIMApiCreatedAnalyticsWidget.jsx:318]: sorry I was talking about the next line(319, 320)

[the remark this answers is not in the document]
Comment (ruwiniwj) [reply components/org.wso2.analytics.apim.widgets/APIMApiCreatedAnalytics/src/APIMApiCreatedAnalyticsWidget.jsx:318]: yes the URL can be moved to a constant.

**Comment by ruwiniwj** [comment]
Fixes https://github.com/wso2/analytics-apim/issues/1144

**Pull request merged**